src/index.html

```diff
@@ -1,27 +1,469 @@
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="Home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#Apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Laptops& Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#TABLETS"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Photo"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Video"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac photo"
+            class="card__photo"
+          />
+        </div>
+        <div class="card__info">
+          <div class="card__name">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__grade">Reviews: 5</div>
+          </div>
+          <div class="card__prices">
```

Comment:
```suggestion
          </div>

          <div class="card__prices">
```
it would be easier to read with blank lines between sibling elements